opentracing-shim/src/main/java/io/opentelemetry/opentracingshim/ScopeManagerShim.java

```diff
@@ -44,7 +44,12 @@ public Span activeSpan() {
 
   @Override
   @SuppressWarnings("MustBeClosedChecker")
-  public Scope activate(Span span) {
+  public Scope activate(@Nullable Span span) {
+    if (span == null) {
+      SpanShim spanShim =
```

Comment:
I think we could cache a final private `SpanShim` instance that simply wraps OTel `Span.getInvalid()`, and we can re-use that for all invocations:

```java
if (span == null) {
  return new ScopeShim(Context.current().with(NOP_SPANSHIM).makeCurrent());
}
```